### ISSUE TITLE
Running Webgoat on Linux RHEL4 Apache/Tomcat gives 401 unauthorized.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kipyegonmark/webgoat#31]
```
What steps will reproduce the problem?
1. Access the url apache-instance/WebGoat/attack
2.
3.

What is the expected output? What do you see instead?
I'm expecting to see the logon screen. If I go to the Tomcat direct via 
port 8080 it works. I have other apps running on Apache that I can acces 
fine.


What version of the product are you using? On what operating system?
Webgoat 5.2
Linux RHEL4
Apache 2.0.61
Tomcat 5.5.16
Java 1.6.32


Please provide any additional information below.
I have verified that the Apache instance is running fine as is Tomcat. No 
error msgs on startup. I just can't get Webgoat to run via Apache.


```

Original issue reported on code.google.com by `andy.p.s...@gtempaccount.com` on 15 Oct 2009 at 12:36

**Comment by GoogleCodeExporter** [comment]
```
Did you add the WebGoat users/roles to tomcate-users.conf?  How to do this can 
be
found in the FAQ.
```

Original comment by `mayhe...@gmail.com` on 15 Oct 2009 at 12:59
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Yes I did. I also forgot to mention that once I've done a successful signon by 
going 
direct to the Tomcat instance I can then acccess via Apache from the same 
browser 
session.
```

Original comment by `andy.p.s...@gtempaccount.com` on 15 Oct 2009 at 1:17

**Comment by GoogleCodeExporter** [comment]
```
Ok, I found the solution. mod_jk needs updating to version 1.2.26 or later. I 
was 
running 1.2.15 and this is a bug fixed in 1.2.26
```

Original comment by `andy.p.s...@gtempaccount.com` on 19 Oct 2009 at 3:42

**Comment by GoogleCodeExporter** [comment]
Original comment by `mayhe...@gmail.com` on 19 Oct 2009 at 5:09
- Changed state: **Done**
